deploy/scripts/src/mobile/utils/README

```diff
@@ -1 +1,38 @@
-url.js from https://jslib.k6.io/url/1.0.0/index.js
\ No newline at end of file
+# DCMAW Performance Testing Framework
```

Comment:
Conscious the folder name is 'mobile' so we probably want to reflect that here, rather than DCMAW